oggm/utils.py

```diff
@@ -3339,13 +3339,32 @@ def write_monthly_climate_file(self, time, prcp, temp,
 
             timev = nc.createVariable('time', 'i4', ('time',))
             tatts = {'units': time_unit}
-            if calendar is not None:
-                tatts['calendar'] = calendar
-                numdate = netCDF4.date2num([t for t in time], time_unit,
-                                           calendar=calendar)
-            else:
-                numdate = netCDF4.date2num([t for t in time], time_unit)
+            timev.long_name = 'time'
+            timev.standard_name = 'time'
+
+
+            ##################################################################################
+            ### ISSSUE: Why converting date to numbers ?
```

Comment:
I don't understand why you needed to change something here, will have to have a look in more detail. But in principle the baseline climate will remain CRU, GCM will never be a baseline climate

Reply:
Yeah, the baseline climate will remain CRU, so maybe this is not the best place to change that. This change was because the time format in CRU netcdf and CMIP5 GCM netcdf were not the same .